netplan-configure.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Netplan Backend Configuration
+Documentation=man:netplan(8)
+Documentation=https://netplan.readthedocs.io/en/stable/netplan-yaml/
+
+DefaultDependencies=no
+Conflicts=shutdown.target
+Before=shutdown.target
+Before=network-pre.target network.target network-online.target nss-lookup.target
+Before=systemd-networkd.service NetworkManager.service systemd-udevd.service
+# Maybe "After=systemd-remount-fs.service" instead?
+After=local-fs.target
```

Comment:
Consider if we need to order `After=systemd-userdbd.service` due to https://bugs.launchpad.net/ubuntu/+source/netplan.io/+bug/2071747